content/experiments/quantum-dice.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Quantum Dice
+author: Russell Huffman
+description: A simple dice roller app using quantum computing to generate random numbers.
+image: https://github.com/JRussellHuffman/quantum-dice/blob/master/assets/header-image.pngs
+to: /experiments/quantum-dice
+media:
+  - https://github.com/JRussellHuffman/quantum-dice/blob/master/assets/header-image.pngs
```

Comment:
Same here. Be careful with the `s` in `pngs`. I think the extension is just `png`.